docs/content/development/extensions-contrib/opentsdb-emitter.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: doc_page
+---
+
+# Opentsdb Emitter
+
+To use this extension, make sure to [include](../../operations/including-extensions.html) `opentsdb-emitter` extension.
+
+## Introduction
+
+This extension emits druid metrics to [OpenTSDB](https://github.com/OpenTSDB/opentsdb) over HTTP. And this emitter only emits service metric events to OpenTSDB (See http://druid.io/docs/latest/operations/metrics.html for a list of metrics).
+
+## Configuration
+
+All the configuration parameters for the opentsdb emitter are under `druid.emitter.opentsdb`.
+
+|property|description|required?|default|
+|--------|-----------|---------|-------|
+|`druid.emitter.opentsdb.host`|The host of the OpenTSDB server.|yes|none|
+|`druid.emitter.opentsdb.port`|The port of the OpenTSDB server.|yes|none|
+|`druid.emitter.opentsdb.connectionTimeout`|Connection timeout(in milliseconds).|no|2000|
+|`druid.emitter.opentsdb.readTimeout`|Read timeout(in milliseconds).|no|2000|  
+|`druid.emitter.opentsdb.batchSize`|Number of events to send as one batch.|no|100|
```

Comment:
I think "flushThreshold" would be a more accurate name instead of "batchSize" since the sender will drain the entire event queue when that value is crossed

Reply:
Sounds good. Fixed.